data/postgres-seed.js

```diff
@@ -0,0 +1,40 @@
+const { db, pgp } = require('./postgres/index.js');
+const earnings = require('./earnings.js');
+const tickers = require('./tickers.js');
+
+const cs = new pgp.helpers.ColumnSet(['ticker', 'name', 'earnings'], { table: 'tickers' });
+
+const start = () => {
+  let tickerList = tickers.createTickers().map(ticker => {
```

Comment:
whenever you call the new createTickers function you're generating 1000, so that acting like your batch?

Reply:
Yes